app/controllers/edit_merchant_details/post-edit-controller.js

```diff
@@ -6,6 +6,7 @@ const paths = require('../../paths')
 const serviceService = require('../../services/service_service')
 const { isPhoneNumber, isValidEmail } = require('../../browsered/field-validation-checks')
 const formattedPathFor = require('../../utils/replace_params_in_path')
+const { validPaths, validOps, ServiceUpdateRequest } = require('../../services/ServiceUpdateRequest.class')
 
 const MERCHANT_NAME = 'merchant-name'
```

Comment:
It might be useful in a future refactor to think about calling these something along the lines of `clientFormIds` or something. This would make it easy to understand what they are when reading the code.

Also it would then mean that
```js
requestFormBody[clientFormsIds.MERCHANT_NAME]
```

would very clearly mean the merchant name attribute from the client submitted body.

Reply:
👍 I'm going to refactor this class a bit at the end of the story which is to add a new page doing similar things to this page, so I'll do that.